src/test/resources/com/powsybl/config/test/config.yml

```diff
@@ -0,0 +1,11 @@
+dynawaltz:
+  homeDir: /home/runner/work/dynamic-simulation-server/dynawo
+  debug: true
+
+dynawaltz-default-parameters:
+  parametersFile: /home/phamquy/Projects/dynawo/test/models.par
```

Comment:
is this used ?

Reply:
No, in fact there is a service, i.e. ParametersService, which configures parameters for Dynawaltz. This service does not use "dynawaltz-default-parameters".
I will remove all params in "dynawaltz-default-parameters"

Reply:
Done